src/common/options/osd.yaml.in

```diff
@@ -520,6 +520,64 @@ options:
     stats (inc. scrub/block duration) every this many seconds.
   default: 120
   with_legacy: false
+- name: osd_scrub_retry_delay
+  type: int
+  level: advanced
+  desc: Period (in seconds) before retrying a specific PG following a scrub failure
```

Comment:
nit: This to me reads as though the option is independently applied to each PG. Suggest

> desc: Period (in seconds) before retrying a PG that has failed a prior scrub.

Reply:
fixed